### ISSUE TITLE
RD had an error, #2

### ISSUE THREAD

**Issue by daFreeMan** [rubberduck-vba/Rubberduck#2099]
I hit <kbd>Ctrl</kbd><kbd>M</kbd> to indent a module and got a RD error.

[rubberduck.log.txt](https://github.com/rubberduck-vba/Rubberduck/files/373902/rubberduck.log.txt)

**Comment by Vogel612** [comment]
Relevant stacktrace from inside the log:
```
2016-07-20 10:23:04.8895;ERROR;DynamicInjectorce6ff4e92e28401192dcbfde42d3c26e;Ninject.ActivationException: Error activating IInspection using binding from IInspection to method
A cyclical dependency was detected between the constructors of two services.

Activation path:
  3) Injection of dependency IInspection into parameter inspections of constructor of type ConfigurationLoader
  2) Injection of dependency IGeneralConfigService into parameter configService of constructor of type TestExplorerViewModel
  1) Request for TestExplorerViewModel

Suggestions:
  1) Ensure that you have not declared a dependency for IInspection on any implementations of the service.
  2) Consider combining the services into a single one to remove the cycle.
  3) Use property injection instead of constructor injection, and implement IInitializable
     if you need initialization logic to be run after property values have been injected.

   at Ninject.Activation.Context.Resolve()
   at System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   at System.Linq.Enumerable.<CastIterator>d__94`1.MoveNext()
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Collections.Generic.HashSet`1.UnionWith(IEnumerable`1 other)
   at System.Collections.Generic.HashSet`1..ctor(IEnumerable`1 collection, IEqualityComparer`1 comparer)
   at Rubberduck.Settings.CodeInspectionConfigProvider.GetDefaultCodeInspections() in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\RetailCoder.VBE\Settings\CodeInspectionConfigProvider.cs:line 47
   at Rubberduck.Settings.CodeInspectionConfigProvider.Create(IEnumerable`1 inspections) in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\RetailCoder.VBE\Settings\CodeInspectionConfigProvider.cs:line 29
   at Rubberduck.Settings.ConfigurationLoader.LoadConfiguration() in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\RetailCoder.VBE\Settings\ConfigurationLoader.cs:line 55
   at Rubberduck.Root.RubberduckModule.<Load>b__8_1() in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\RetailCoder.VBE\Root\RubberduckModule.cs:line 91
   at Rubberduck.SmartIndenter.Indenter.BuildLogicalCodeLines(IEnumerable`1 lines) in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\Rubberduck.SmartIndenter\Indenter.cs:line 123
   at Rubberduck.SmartIndenter.Indenter.Indent(IEnumerable`1 codeLines, String moduleName, Boolean reportProgress, Int32 linesAlreadyRebuilt) in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\Rubberduck.SmartIndenter\Indenter.cs:line 150
   at Rubberduck.SmartIndenter.Indenter.Indent(VBComponent component, Boolean reportProgress, Int32 linesAlreadyRebuilt) in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\Rubberduck.SmartIndenter\Indenter.cs:line 88
   at Rubberduck.UI.Command.CommandBase.Execute(Object parameter) in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\RetailCoder.VBE\UI\Command\CommandBase.cs:line 60;
```

**Comment by Vogel612** [comment]
closed in #2115
